api/upgradeseries/upgradeseries.go

```diff
@@ -59,10 +59,41 @@ func (s *Client) MachineStatus() (model.UpgradeSeriesStatus, error) {
 	if params.IsCodeNotFound(r.Error) {
 		return "", errors.NewNotFound(r.Error, "")
 	}
-
 	return "", errors.Trace(r.Error)
 }
 
+// UnitsReadyToStop returns the units running on this machine that have
+// completed their upgrade-series preparation, and are ready to be stopped and
+// have their unit agent services converted for the target series.
+func (s *Client) UnitsReadyToStop() ([]names.UnitTag, error) {
```

Comment:
I will rename this to something more general, as it is not actually used just to detect that units are ready to stop.